pkg/cmd/openshift-tests/images/images_command.go

```diff
@@ -151,7 +155,36 @@ type imagesOptions struct {
 func createImageMirrorForInternalImages(prefix string, ref reference.DockerImageReference, mirrored bool) ([]string, error) {
 	source := ref.Exact()
 
+	// This is coming from the vendored directory
 	initialDefaults := k8simage.GetOriginalImageConfigs()
+
+	// If ENV is set, this should come from external binary
+	if len(os.Getenv("OPENSHIFT_SKIP_EXTERNAL_TESTS")) == 0 {
+		// Extract all test binaries
+		extractLogger := log.New(os.Stdout, "", log.LstdFlags|log.Lmicroseconds)
+		extractionContext, extractionContextCancel := context.WithTimeout(context.Background(), 30*time.Minute)
+		defer extractionContextCancel()
+		cleanUpFn, externalBinaries, err := externalbinary.ExtractAllTestBinaries(extractionContext, extractLogger, 10)
+		if err != nil {
+			return nil, err
+		}
+		defer cleanUpFn()
+
+		// List test images from all available binaries and convert them to origin's testCase format
+		listContext, listContextCancel := context.WithTimeout(context.Background(), time.Minute)
+		defer listContextCancel()
+
+		imagesFromExternalBinaries, err := externalBinaries.ListImages(listContext, 10)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(imagesFromExternalBinaries) > 0 {
+			// FIXME: only 1 for now
+			initialDefaults = imagesFromExternalBinaries[0]
+		}
```

Comment:
It'd be better if we could gather up all the external image binaries here

Reply:
My latest version implements this